functions/src/client/ping.ts

```diff
@@ -0,0 +1,18 @@
+import { get } from 'request-promise-native';
+
+export interface PingResponse {
+  statusCode: number;
+  body: string;
+}
+
+export async function Ping<T>(uri: string, token: string): Promise<T> {
+  return <T> await get({
+    uri: uri,
+    headers: {
+      'User-Agent': 'DashboardHub',
+      'Authorization': `token ${token}`,
```

Comment:
I dont think `Authorization` is needed? It will be pinging public URLs that are reachable in the browser by anyone

Reply:
okay let me check. it was not working without token earlier.

Reply:
great catch. I have removed the authorization . It worked.